week3/homework/src/app.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const utilities = require('./utilities');
```

Comment:
I would change the `utilities` file name to something like `api`, since it's very related to the api you have.

utilities is more generic name, that contains functions and methods about many things and can be used in many places! such as what you have in `data_dealer.js` file :)
which leads to `data_dealer.js` could be the `utilities.js`?

Reply:
At that moment, I ran out of names. 
You understood my files/folders structure and your suggestion is perfect with it.